client/pages/form-pages/support-docs-checklist.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useCallback } from 'react';
+import { Form, Formik, FormikProps } from 'formik';
+import { FormStepTitles, FormSteps, FileUpload } from '@components';
+import { GeneralInfoInterface } from 'constants/interfaces';
+
+const initialValues = {
+  estimatedCost: '',
```

Comment:
This isn't updated for this form.

Reply:
Updated initial values to this form. Will have to revisit when page gets saved.